### PR TITLE
Change dependency due to moving org mgt service to org mgt core

### PR DESCRIPTION
**Pull request by vinumaddumage**
### Proposed changes in this pull request

dependency changes due to moving org mgt service to org mgt core
version bump

Merge after:
https://github.com/wso2-extensions/identity-organization-management/pull/136
https://github.com/wso2/carbon-identity-framework/pull/4215

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -830,10 +826,15 @@
         <carbon.identity.framework.imp.pkg.version.range>[5.17.5, 6.0.0)
         </carbon.identity.framework.imp.pkg.version.range>
 
-        <carbon.identity.organization.management.version>1.0.54</carbon.identity.organization.management.version>
+        <carbon.identity.organization.management.version>1.1.5</carbon.identity.organization.management.version>
         <carbon.identity.organization.management.version.range>[1.0.46, 2.0.0)
```
Comment (wenodpathirana) [inline pom.xml:830]: Doesn't lower bound have to be changed too?

File: pom.xml [hunk 2]
```diff
@@ -237,20 +237,16 @@
                 <version>${carbon.identity.framework.version}</version>
             </dependency>
             <dependency>
-                <groupId>org.wso2.carbon.identity.organization.management</groupId>
+                <groupId>org.wso2.carbon.identity.organization.management.core</groupId>
                 <artifactId>org.wso2.carbon.identity.organization.management.service</artifactId>
-                <version>${carbon.identity.organization.management.version}</version>
-                <exclusions>
-                    <exclusion>
```
Comment (wenodpathirana) [inline pom.xml:244]: looks like following comes from the framework dependency which will cause cyclic dependency, we will have to upgrade the identity framework version or exclude org management from the dep.
```
\- org.wso2.carbon.identity.organization.management:org.wso2.carbon.identity.organization.management.service:jar:1.0.65:compile
     \- org.wso2.carbon.identity.organization.management:org.wso2.carbon.identity.organization.management.authz.service:jar:1.1.5:compile
        \- org.wso2.carbon.identity.auth.rest:org.wso2.carbon.identity.authz.service:jar:1.6.2-SNAPSHOT:compile
           \- org.wso2.carbon.identity.auth.rest:org.wso2.carbon.identity.auth.service:jar:1.6.2-SNAPSHOT:compile
```

**Review by wenodpathirana — APPROVED** [review]

**Review by AnuradhaSK — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (vinumaddumage) [reply pom.xml]: updated

File: pom.xml (hunk 2 above)
Comment (vinumaddumage) [reply pom.xml:244]: upgraded framework version

**Pull request merged**